test/parallel.jl

```diff
@@ -1,8 +1,12 @@
+
+cd(cwd)
```

Comment:
What is `cwd`?

Reply:
if the `io.jl` file ran correctly, `cwd ` is the `test` directory.
but i can remove this if you want